wordpress-mobile/gutenberg-mobile
# Show "Add to beginning" and "Add to end" option for all blocks when long-pressing the inserter button

Fixes #2344 

### Related PR
`gutenberg`: https://github.com/WordPress/gutenberg/pull/28610

To test:
1. Open the demo app
2. Long-press on the inserter button, verify options **Add to beginning** and **Add to end** are displayed. 
3. Tap on **Add to beginning** and select a block. Verify block added to the beginning. 
4. With no block selected, long-press on inserter again, this time tap on **Add to end** and select a block. Verify added to end of post. 
5. Select a block in the editor, then long-press on the inserter. Verify new options **Add to beginning** and **Add to end** are displayed.
6. Tap on **Add to beginning** and select a block. Verify added to beginning of document. 
7. Repeat for the other options available when long-pressing on inserter with a block selected and verify each work. 

PR submission checklist:
